content/docs/notification/notifications.md

```
@@ -0,0 +1,163 @@
+title: Notifications
+---
+
+# 💬🔔 Notifications
+
+This plugin allows sending messages to different parts of Minecraft, including the chat, action bar, title, and subtitle. All messages can be formatted using [🔍 MiniMessages](https://docs.adventure.kyori.net/minimessage/format.html) or standard color codes. You can also use an online [🌐 message viewer](https://webui.adventure.kyori.net/)
+
+<Alert type="tip" title="TIP">
+Try our [interactive notification generator](/notification-generator) to create and preview your notifications before adding them to your configuration!
+</Alert>
+
+### 📝 Example usage
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)